play.pokemonshowdown.com/js/client-teambuilder.js

```diff
@@ -3680,7 +3687,61 @@
 			}
 		}
 	});
+	var DingbatPopup = exports.DingbatPopup = Popup.extend({
+		initialize: function (data) {
+			var dingbats_halfwidth = [
+				"&#xe095;","&#xe096;","&#xe097;","&#xe098;",
+				"&#xe099;","&#xe090;","&#xe091;","&#xe092;",
+				"&#xe093;","&#xe094;","&#xe09a;",
+				"\n",
+				"&#xe09b;","&#xe09c;","&#xe09d;","&#xe09e;",
+				"&#xe09f;","&#xe0a0;","&#xe0a1;","&#xe0a2;",
+				"&#xe0a5;","&#xe0a3;","&#xe0a4;",	
+				"\n",
+				"&#xe08e;","&#xe08f;"
+			];
+			var dingbats_fullwidth = [
+				"&#x25CE;", "&#x25CB;", "&#x25A1;", "&#x25B3;", "&#x25C7;",
+				"&#x2660;", "&#x2663;", "&#x2665;", "&#x2666;", "&#x2605;", "&#x266A;", 
+				"\n",
+				"&#x2600;", "&#x2601;", "&#x2602;", "&#x2603;",
+				"&#xE081;", "&#xE082;", "&#xE083;", "&#xE084;", "&#xE087;", "&#xE085;", "&#xE086;",
+				"\n",
+				"&#x2642;","&#x2640;"
+			];
+			this.i = data.i;
+			this.team = data.team;
 
+			var buf = '<ul name="dingbats" class="popupmenu"><li>';
+			for (var i=0; i<dingbats_halfwidth.length; i++) {
+				if (dingbats_halfwidth[i] == '\n'){
+					buf +='</li><li>'
+				} else {
+					buf += '<button name="insertDingbat" class="dingbat-button"';
+					buf += " halfwidth=" + dingbats_halfwidth[i] 
+					buf += " fullwidth=" + dingbats_fullwidth[i];
+					buf += " value=" + dingbats_halfwidth[i];
+					buf += ">" + dingbats_halfwidth[i] + "</button>";
+				}
+			}
+			buf += '</li><input type="checkbox" name="wideCheckbox"></input>Wide</ul>';
+			this.$el.html(buf);
+			var el=this.$el;
```

Comment:
We generally would name this variable `$el`, like we do for other jQuery objects.